ckcc/cli.py

```diff
@@ -1010,4 +1006,97 @@ def get_storage_locker():
 
     click.echo(ls)
 
+
+keystore_keys = ["derivation", "hw_type", "label", "root_fingerprint", "soft_device_id", "xpub"]
+
+@main.command('coldcardify')
+@click.argument('file', type=click.Path(exists=True), required=True)
+@click.option('--outfile', '-o', type=click.Path(),
+              help="output file path where adjusted wallet file is written. "
+                   "If this is not specified output is written to <original_file>_cc")
+@click.option('--dry-run', '-n', default=False, is_flag=True, help="do not write files instead pretty print to console")
+@click.option('--key', '-k', type=click.Choice(keystore_keys),
```

Comment:
key/val is probably too versatile, but I did not want to prohibit this to only hw_type (as one can have multiple from same vendor) or fingerprint...